### PR TITLE
Translate "compute shader" and "shader" to Ukrainian

### PR DESCRIPTION
**Pull request by Herobread**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Oleksii Nazarenko

## Language: 

- Ukrainian

## Terms defined:

- compute shader
- shader

### CODE REVIEW

**Review by olexandr-konovalov — COMMENTED** [review]
Would be nice to extend this PR with translation of shader as well!

**Comment by Herobread** [comment]
> Would be nice to extend this PR with translation of shader as well (hope that was the initial plan!)

of course, working on it!

(secretly updates original plan using git --amend)

**Comment by olexandr-konovalov** [comment]
Just in case, note https://github.com/carpentries/glosario/pull/644

File: glossary.yml [hunk 1]
```diff
@@ -9119,6 +9125,12 @@
     def: >
       A program designed to run on the [GPU][gpu]. Generally used in graphics to calculate lighting or position vertices
       in a scene, though can be used for more general programming through the use of [compute shaders][#compute_shader].
+  uk:
+    term: "шейдер"
+    def: >
+      Програма, призначена для роботи на [GPU](#gpu). Загалом, використовується у графіці для розрахунку освітлення
```
Comment (olexandr-konovalov) [inline glossary.yml:9131]: для роботи ->  для виконання ?

File: glossary.yml [hunk 2]
```diff
@@ -3093,6 +3093,12 @@
       A general purpose [shader][#shader] program for use in parallel processing. Often used
       for [machine learning][#machine_learning], simulations, and other fields which benifit
       from parallel computation.
+  uk:
+    term: "обчислювальні шейдери"
```
Comment (olexandr-konovalov) [inline glossary.yml:3097]: Why do you use plural here, if singular is used otherwise?

Comment (Herobread) [reply glossary.yml]: @olexandr-konovalov I have changed to singular now

**Comment by olexandr-konovalov** [comment]
@Herobread you need to resolve conflicts - @YehorBoiar or I could help, better in a join videocall

**Comment by Herobread** [comment]
@olexandr-konovalov I have resolved the conflicts

File: glossary.yml (hunk 2 above)
Comment (olexandr-konovalov) [reply glossary.yml:3047]: @Herobread hm, I still see обчислювальні шейдери in plural

**Comment by olexandr-konovalov** [comment]
LGTM! 2nd reviewer needed (@YehorBoiar or @VeronikaShevc ?)

**Review by YehorBoiar — APPROVED** [review]

**Review by olexandr-konovalov — APPROVED** [review]
@elletjies please merge

**Pull request merged**